packages/app-builder-backend/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@reapit/app-builder-backend",
+  "version": "0.0.1",
+  "description": "Common Package for Reapit Foundation",
+  "repository": {
+    "type": "git",
+    "url": "git+git@github.com:reapit/foundations.git"
+  },
+  "license": "MIT",
+  "author": "Reapit Ltd",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start": "ts-node .",
+    "test": "echo '...skipping...'",
+    "build": "tsc",
```

Comment:
Does this include your node modules correctly? Great if we can get away with no webpack

Reply:
yep! the node modules get bundled into the docker container on build
https://github.com/reapit/foundations/blob/feat/%234523-app-builder-backend/Dockerfile#L50